pkg/virt-controller/watch/vm.go

```diff
@@ -226,8 +230,17 @@ func (c *VMController) execute(key string) error {
 			return nil
 		}
 
+		// Ensure registry disks are online before placing VM
+		if registrydisk.DisksAreReady(&pods.Items[0]) == false {
+			return errors.New("waiting on image wrapper disks to become ready")
```

Comment:
In that case the appropriate thing would be to log that we are waiting for the image to be ready. It can take some time. It is not an error, therefore we need no automatic re-queue. After all disks are ready, we get notified by the apiserver because of the pod state change. I would log it as info with a severity > 1 (we should somwhere define what log level to use for what ...) and return nil.